11-payment-encoding.md

```diff
@@ -109,6 +109,7 @@ Currently defined Tagged Fields are:
 * `n` (19): `data_length` 53.  The 33-byte public key of the payee node.
 * `h` (23): `data_length` 52.  256-bit description of purpose of payment (SHA256).  This is used to commit to an associated description which is too long to fit, such as may be contained in a web page.
 * `x` (6): `data_length` variable.  `expiry` time in seconds (big-endian). Default is 3600 (1 hour) if not specified.
+* `c` (24): `data_length` (16 bits, big-endian). Minimum `cltv_expiry` to use for the last htlc. Default is 9 if not specified.
```

Comment:
I think the `data_length` section here should be modified to be 4 (4 * 5 = 20, so 4 unused bits).

Reply:
Make it variable, like x.  If you can fit it in 5 bits, great.  We should enforce a maximum of 6 (ie. fits in 32 bits) though, that's a separate PR.

Reply:
Can we rename it to `final_cltv` or `min_final_cltv` because we already have a `cltv_expiry` field in the onion...

Reply:
In the `update_channel` messages the delta is expressed using 16 bits. Seem like we should maintain uniformity of the allotted size of "delta" values throughput the spec.

Agreed on using a distinct name.

Reply:
Ack...

Reply:
Yes, I chose 16 bits to be consistent with `update_channel` as laolu points out. Will update the name.